go/chat/flip/prot.avdl

```diff
@@ -0,0 +1,101 @@
+
+@namespace("flip")
```

Comment:
Anyway we could make this part of `chat1`? I can also just do that when integrating.

Reply:
sure, we can.  it was nice to have it all self-contained.  but maybe that's silly

Reply:
BTW, these prot types are actually never explicit exported. They are all hidden from chat since chat is just passing around opaque messages.

Reply:
Eh, it's unfortunately a big change...

Reply:
What if I leave it as is?

Reply:
Why is there AVDL at all?